### PR TITLE
Add support for rt1019_rt5682 for RPL

### PR DESCRIPTION
**Pull request by terry182**
Boards were using this in older kernels before adl and rpl ids were split. Add this back to maintain support.

### CODE REVIEW

File: sound/soc/intel/boards/sof_rt5682.c [hunk 1]
```diff
@@ -1144,6 +1144,17 @@ static const struct platform_device_id board_ids[] = {
...
+		.name = "rpl_rt1019_rt5682",
+		.driver_data = (kernel_ulong_t)(SOF_RT5682_MCLK_EN |
+					SOF_RT5682_SSP_CODEC(0) |
+					SOF_SPEAKER_AMP_PRESENT |
+					SOF_RT1019_SPEAKER_AMP_PRESENT |
+					SOF_RT5682_SSP_AMP(1) |
+					SOF_RT5682_NUM_HDMIDEV(4) |
+					SOF_BT_OFFLOAD_SSP(2) |
+					SOF_SSP_BT_OFFLOAD_PRESENT),
+	},
```
Comment (plbossart) [inline sound/soc/intel/boards/sof_rt5682.c]: can you move this up to line 1122 to keep all RPL descriptor together? Thanks!

**Review by plbossart — CHANGES_REQUESTED** [review]
nit-pick below

File: sound/soc/intel/boards/sof_rt5682.c (hunk 1 above)
Comment (terry182) [reply sound/soc/intel/boards/sof_rt5682.c]: No problem at all. I wonder should I submit another patch to re-order things, as I saw some JSL and TGL scattered.

Comment (plbossart) [reply sound/soc/intel/boards/sof_rt5682.c]: that would be great @terry182, this file is a bit of a mess. Thanks!

**Review by cujomalainey — APPROVED** [review]

**Review by plbossart — APPROVED** [review]

**Pull request merged**